docs/reference/authenticated_authors.rst

```diff
@@ -12,13 +12,10 @@ This is a ``FOSCommentBundle`` feature. We will explain to you here how to enabl
 Implement SignedCommentInterface
 --------------------------------
 
-Open entity ``Application\Sonata\CommentBundle\Entity\Comment`` and implement the following methods and interface:
+Open entity ``App\Application\Sonata\CommentBundle\Entity\Comment``
+and implement the following methods and interface::
 
-.. code-block:: php
-
-    <?php
-
-    namespace Application\Sonata\CommentBundle\Entity;
+    namespace App\Application\Sonata\CommentBundle\Entity;
```

Comment:
The namespace should be `App\Entity` only

See the flex code https://github.com/symfony/recipes-contrib/blob/master/sonata-project/media-orm-pack/1.0/src/Entity/SonataMediaGallery.php#L5

Reply:
so i need to fix it everywhere, right?

an I can fully remove the easy extends stuff?

Reply:
> an I can fully remove the easy extends stuff?

👍

Reply:
I double checked, but we don't have a recipe for comment bundle 🤔 

https://github.com/symfony/recipes-contrib/tree/master/sonata-project

Reply:
ping @core23